ramls/inventory/classificationtype.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "A classification type",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "string"
```

Comment:
Maybe it makes sense to add description to make it in the same style as other fields

Reply:
"$ref": "uuid.json"?